hack/remove-calico-policy/system-network-policy-override.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "alpha.projectcalico.org/v1"
+kind: SystemNetworkPolicy
+metadata:
+  name: allow-all-calico-policy
```

Comment:
I just copied the original manifest, but I think it's better to make namespace part of the manifest. will do that